soc/arm/nxp_lpc/lpc55xxx/Kconfig.soc

```diff
@@ -53,6 +53,7 @@ config SOC_LPC55S36
 	select ARMV8_M_DSP
 	select ARM_TRUSTZONE_M
 	select CLOCK_CONTROL
+	select HAS_MCUX_LPADC
```

Comment:
This is unnecessary to select here, it will be enabled by default because of [devicetree kconfig macros](https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/adc/Kconfig.mcux), better to have less selects across the tree to have less points to maintain